test/metabase/test/data/mbql_query_impl.clj

```diff
@@ -1,12 +1,19 @@
 (ns metabase.test.data.mbql-query-impl
   "Internal implementation of `data/$ids` and `data/mbql-query` and related macros."
   (:require [clojure.string :as str]
+            [clojure.test :refer :all]
             [clojure.walk :as walk]
             [metabase.models.field :refer [Field]]
             [toucan.db :as db]))
 
+(declare parse-token-by-sigil)
```

Comment:
couldn't you just bring `parse-token-by-sigil` up in the file?

Reply:
good point. I'll change it